src/platform-includes/performance/add-spans-example/go.mdx

```diff
@@ -1,15 +1,16 @@
 ## Add More Spans to the Transaction
 
-The next example contains the implementation of the hypothetical `doWork` function called from the code snippet in the previous section. Our SDK can determine if there is currently an open transaction in the current context and add all newly created spans as child operations to that transaction. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the transaction.
-
-You can choose the value of `operation` and `description`.
+The next example contains the implementation of the hypothetical `doWork` function called from the code snippet in the previous section. Our SDK can determine if there is currently an open transaction in the current context and add all newly created spans as child operations to that transaction. Keep in mind that each individual span also needs to be manually finished. Otherwise, spans will not show up on the transaction.
 
 ```go
-func doWork(ctx context.Context, item Item) {
-	span := sentry.StartSpan(ctx, "suboperation1")
+func doWork(ctx context.Context) {
+	span := sentry.StartSpan(ctx, "function")
+	span.Description = "suboperation1"
```

Comment:
How about this:

```suggestion
	span := sentry.StartSpan(ctx, "span name 1")
	span.Description = "span description 1"
```
(and same below)

Otherwise "function" is a bit too specific (we don't only wrap functions in spans), and also both spans will be named the same.

Reply:
name -> operation

Reply:
The `op` should be from https://develop.sentry.dev/sdk/performance/span-operations/#general; more context is added with the description.

Reply:
Ah, confused with StartTransaction. 
Let's then link to https://develop.sentry.dev/sdk/performance/span-operations/ here? This is the part that would be confusing to me personally, it's not immediately clear what should go into that second argument.